api/admin/controller/metadata_services.py

```diff
@@ -107,17 +109,15 @@ def validate_form_fields(self, **fields):
         if wrong_format:
             return wrong_format
 
-    def register_with_metadata_wrangler(self, is_new, service):
+    def register_with_metadata_wrangler(self, do_get, do_post, is_new, service):
         """Register this site with the Metadata Wrangler."""
 
         if ((is_new or not service.password) and
             service.protocol == ExternalIntegration.METADATA_WRANGLER):
 
-            problem_detail = self.sitewide_registration(
-                service, do_get=do_get, do_post=do_post
```

Comment:
This is the core problem. The method name is process_sitewide_registration, not sitewide_registration, and the arguments being passed in -- do_get and do_post -- were not being propagated from process_post into register_with_metadata_wrangler.